### ISSUE TITLE
Literal not working when applied for <svg>  <g display=${noshow}> 

### ISSUE THREAD

**Issue by vikjung** [Polymer/lit-html#469]
Literal not working when applied for <svg>  <g display=${noshow}>   but works if use class < g class ="noshow" > . 

**following does not work** 
with either display=${dikhao2}  or .display=${dikhao3} or display=${dikhao2} or .display=${dikhao3}
but can make it work if use the literals in the class and then use in svg

----
const dikhao2= html backtick none backtick;
const dikhao3= "none";

`<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="24px" height="24px" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
<g id="Header_x2F_BG" display="none">
	<rect x="-402" y="-50" display="inline" fill="#F1F1F2" width="520" height="520"/>
</g>
<g id="Bounding_Boxes">
	<g id="ui_x5F_spec_x5F_header_copy_3">
	</g>
	<path fill="none" d="M0,0h24v24H0V0z"/>
</g>
<g id="Outline" display=${dikhao2}>  
	<g id="ui_x5F_spec_x5F_header">
	</g>
	<g>
		<path d="M12,2C6.48,2,2,6.48,2,12c0,5.52,4.48,10,10,10c5.52,0,10-4.48,10-10C22,6.48,17.52,2,12,2z M7.07,18.28c0.43-0.9,3.05-1.78,4.931.78s4.51,0.88,4.93,1.78C15.57,19.36,13.86,20,12,20S8.43,19.36,7.07,18.28z M18.36,16.83c-1.43-1.74-4.9-2.33-6.36-2.33s-4.93,0.59-6.36,2.33C4.62,15.49,4,13.82,4,12c0-4.41,3.59-8,8-8c4.41,0,8,3.59,8,8C20,13.82,19.38,15.49,18.36,16.83z"/>
<path d="M12,6c-1.94,0-3.5,1.56-3.5,3.5S10.06,13,12,13c1.94,0,3.51.56,3.53.5S13.94,6,12,6zM12,11c-0.83,0-1.5-0.671.51.5C10.5,8.67,11.17,8,12,8c0.83,0,1.5,0.67,1.5,1.5C13.5,10.33,12.83,11,12,11z"/>
	</g>
</g>
<g id="nyt_x5F_exporter_x5F_info" display="none">
</g>
</svg>
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="24px"
	 height="24px" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve">
<g id="Header_x2F_BG" display="none">
	<rect x="-402" y="-50" display="inline" fill="#F1F1F2" width="520" height="520"/>
</g>
<g id="Bounding_Boxes">
	<g id="ui_x5F_spec_x5F_header_copy_3">
	</g>
	<path fill="none" d="M0,0h24v24H0V0z"/>
</g>
<g id="Fill" .display=${dikhao2}>  
	<g id="ui_x5F_spec_x5F_header_copy" display="inline">
	</g>
	<path display="inline" d="M12,2C6.48,2,2,6.48,2,12s4.48,10,10,10s10-4.48,10-10S17.52,2,12,2z M12,5c1.66,0,3,1.34,3,3
		s-1.34,3-3,3S9,9.66,9,8S10.34,5,12,5z M12,19.2c-2.5,0-4.71-1.28-6-3.22c0.03-1.99,4-3.08,6-3.08c1.99,0,5.97,1.09,6,3.08
		C16.71,17.92,14.5,19.2,12,19.2z"/>
</g>
<g id="nyt_x5F_exporter_x5F_info" display="none">
</g>
</svg>`

**following works** 

const dikhao2= html backtick none backtick;
const dikhao3= "none";

      .dikhao2 {
        display : ${dikhao2};
      }
      .dikhao3 {
        display : ${dikhao3};
      }<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="24px"
	 height="24px" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve"><g id="Header_x2F_BG" display="none">
	<rect x="-402" y="-50" display="inline" fill="#F1F1F2" width="520" height="520"/></g><g id="Bounding_Boxes">
	<g id="ui_x5F_spec_x5F_header_copy_3">
	</g>
	<path fill="none" d="M0,0h24v24H0V0z"/></g><g id="Outline" class="dikhao2">  
	<g id="ui_x5F_spec_x5F_header">
	</g>
	<g><path d="M12,2C6.48,2,2,6.48,2,12c0,5.52,4.48,10,10,10c5.52,0,10-4.48,10-10C22,6.48,17.52,2,12,2z M7.07,18.28
			c0.43-0.9,3.05-1.78,4.93-1.78s4.51,0.88,4.93,1.78C15.57,19.36,13.86,20,12,20S8.43,19.36,7.07,18.28z M18.36,16.83
			c-1.43-1.74-4.9-2.33-6.36-2.33s-4.93,0.59-6.36,2.33C4.62,15.49,4,13.82,4,12c0-4.41,3.59-8,8-8c4.41,0,8,3.59,8,8
			C20,13.82,19.38,15.49,18.36,16.83z"/>
		<path d="M12,6c-1.94,0-3.5,1.56-3.5,3.5S10.06,13,12,13c1.94,0,3.5-1.56,3.5-3.5S13.94,6,12,6z M12,11c-0.83,0-1.5-0.67-1.5-1.5
			C10.5,8.67,11.17,8,12,8c0.83,0,1.5,0.67,1.5,1.5C13.5,10.33,12.83,11,12,11z"/>
	</g></g><g id="nyt_x5F_exporter_x5F_info" display="none"></g></svg><svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="24px"
	 height="24px" viewBox="0 0 24 24" enable-background="new 0 0 24 24" xml:space="preserve"><g id="Header_x2F_BG" display="none">
	<rect x="-402" y="-50" display="inline" fill="#F1F1F2" width="520" height="520"/></g><g id="Bounding_Boxes">
	<g id="ui_x5F_spec_x5F_header_copy_3">
	</g>
	<path fill="none" d="M0,0h24v24H0V0z"/></g><g id="Outline" class="dikhao3">  
	<g id="ui_x5F_spec_x5F_header"></g><g><path d="M12,2C6.48,2,2,6.48,2,12c0,5.52,4.48,10,10,10c5.52,0,10-4.48,10-10C22,6.48,17.52,2,12,2z M7.07,18.28
			c0.43-0.9,3.05-1.78,4.93-1.78s4.51,0.88,4.93,1.78C15.57,19.36,13.86,20,12,20S8.43,19.36,7.07,18.28z M18.36,16.83
			c-1.43-1.74-4.9-2.33-6.36-2.33s-4.93,0.59-6.36,2.33C4.62,15.49,4,13.82,4,12c0-4.41,3.59-8,8-8c4.41,0,8,3.59,8,8
			C20,13.82,19.38,15.49,18.36,16.83z"/><path d="M12,6c-1.94,0-3.5,1.56-3.5,3.5S10.06,13,12,13c1.94,0,3.5-1.56,3.5-3.5S13.94,6,12,6z M12,11c-0.83,0-1.5-0.67-1.5-1.5
			C10.5,8.67,11.17,8,12,8c0.83,0,1.5,0.67,1.5,1.5C13.5,10.33,12.83,11,12,11z"/>
	</g></g><g id="nyt_x5F_exporter_x5F_info" display="none"></g></svg>`

**Comment by daKmoR** [comment]
can you maybe rephrase? I seem to not understand the actual problem?

do you maybe mean that something like this does not work for you?
```
const result = svg`
  <svg>
      // ...
     <g display="${'none'}">
  // ...
`;
```

what version of lit-html are you using? how does your import for render and html look like?

**Comment by vikjung** [comment]
Using - lit-html@0.10.2 

Hope this makes it more clear - 

```
const noneconstant = "none";

const result = html `<svg >
// ...
<g display=${noneconstant}>
// ....
</g>
</svg>`;

class ZatobowlHome extends PageViewElement {
  _render(props) {
    return html`
// ...
    ${result} 
// ...
`}};
```

**Comment by hyperpress** [comment]
I might be wrong, but I believe this was fixed in 0.11.0.

**Comment by vikjung** [comment]
Still there with 0.11.0 unless I am doing something stupid ... I am a part time coder.

**Comment by ruphin** [comment]
I cannot reproduce the problem. This is using the code you posted:

https://codepen.io/ruphin/pen/rZwMaP?editors=0010#0

**Comment by vikjung** [comment]
Yes seems to be fixed with 0.11.1 I will cross check my pwa starter kit setup seems like have older version of lit-html under lit-element folder and that might be why it persisted even when I upgraded lit-html version in the lit-html folder. Will close once confirm after checking with a fresh copy of pwa starter kit.
